calico_versioned_docs/version-3.27/about/product-comparison.mdx

```diff
@@ -55,41 +55,46 @@ What is the best fit for you? It depends on your needs. The following table prov
-| Packet capture                                               |                          | <center>&#10003</center> | <center>&#10003</center> |
-| Elasticsearch  logs (flow, l7, audit, bgp, dns, events)      |                          | <center>&#10003</center> | <center>&#10003</center> |
-| Alerts                                                       |                          | <center>&#10003</center> | <center>&#10003</center> |
+| **Application observability and troubleshooting**            | <center>**Calico Open Source**</center> | <center>**Calico Cloud**</center> | <center>**Calico Enterprise**</center> |
+| Graphical view of deployment (Service Graph)                                |<center>Using third-party tools</center>| <center>&#10003</center> | <center>&#10003</center> |
+| Packet capture                                               |<center>Using third-party tools</center>| <center>&#10003</center> | <center>&#10003</center> |
+| Exportable logs                                              |<center>L3-4 flows, manual integration with any SIEM</center> | Preconfigured Elasticsearch dashboards: flow, audit, bgp, dns, L7 | Preconfigured Elasticsearch dashboards: flow, audit, bgp, dns, L7 |
+| Prometheus for metrics and alert monitoring |<center>Supports manual deployment</center>| <center>&#10003</center> | <center>&#10003</center> |
 | Kibana DNS dashboards                                        |                          | <center>&#10003</center> | <center>&#10003</center> |
 | Traffic Flow Visualizer                                      |                          | <center>&#10003</center> | <center>&#10003</center> |
-| **Multi-cluster management**                                 |                          | <center>&#10003</center> | <center>&#10003</center> |
-| Multi-cluster management                                     |                          |                          | <center>&#10003</center> |
-| Federated identity and services                              |                          | <center>&#10003</center> | <center>&#10003</center> |
-| **Threat defense**                                           |                          |                          |                          |
+| **Kubernetes security posture management** | <center>**Calico Open Source**</center> | <center>**Calico Cloud**</center> | <center>**Calico Enterprise**</center> |
+| Review overall score based on namespace isolation, container image vulnerabilities, and egress access | | <center>&#10003</center> |  |
+| Prioritized list of remediation actions | | <center>&#10003</center> |  |
+| **Cluster mesh**                                             | <center>**Calico Open Source**</center>                      | <center>**Calico Cloud**</center>                            | <center>**Calico Enterprise**</center>                       |
```

Comment:
I think you should remove the whitespace from this line so it lines up better in the raw text.